### PR TITLE
trying to make doc intro more appealing

### PR DESCRIPTION
**Pull request by andrewkern**
i did a touchup to the intro section of the documentation to try to make it a bit more friendly from the get go

### CODE REVIEW

File: docs/introduction.rst [hunk 1]
```diff
@@ -35,11 +35,25 @@ First steps
...
+``stdpopsim`` **is a community effort, and we welcome YOU to join us!**
+
 Are there other features, models or organisms that you'd like to see in ``stdpopsim``?
 This software is maintained by the PopSim Consortium,
-a global community of scientists and developers who are working together to improve
+an open, global community of scientists and developers who are working together to improve
 standards for benchmarking and testing in population genetics.
-Read the :ref:`Development <sec_development>` page to find out how you can join us.
+You can get into contact with the ``stdpopsim`` community by subscribing to our email list
+`serve <https://lists.uoregon.edu/mailman/listinfo/popgen_benchmark>`_
```
Comment (agladstein) [inline docs/introduction.rst:45]: link "email list serve"

```
`email list serve <https://lists.uoregon.edu/mailman/listinfo/popgen_benchmark>`_
```

File: docs/introduction.rst [hunk 2]
```diff
@@ -35,11 +35,25 @@ First steps
...
+and by creating and commenting on
+Github `issue <http://github.com/popgensims/stdpopsim/issues>`_.
```
Comment (agladstein) [inline docs/introduction.rst:47]:
```
`issues <http://github.com/popgensims/stdpopsim/issues>`_.
```

File: docs/introduction.rst [hunk 3]
```diff
@@ -35,11 +35,25 @@ First steps
...
+There is a lot of chatter through
+Github, and we’ve been building code
+there cooperatively. If you are interested in joining us, as we very much hope you
+are, please read our
+`code of conduct <https://github.com/popsim-consortium/stdpopsim/blob/master/CODE_OF_CONDUCT.md>`_.
+A particular goal of this project is to build a community which is representative of population
+and evolutionary genetics writ large.
```
Comment (agladstein) [inline docs/introduction.rst:54]: never heard this phrase before.

**Comment by andrewkern** [comment]
yeah i think we could definitely do that. should i take a stab at it? I'll correct all these other issues and push too.

File: docs/introduction.rst (hunk 3 above)
Comment (andrewkern) [reply docs/introduction.rst:54]: okay will change this phrase

**Comment by agladstein** [comment]
> should i take a stab at it?

Yeah, go for it.

File: docs/introduction.rst [hunk 4]
```diff
@@ -41,17 +41,20 @@ Are there other features, models or organisms that you'd like to see in ``stdpop
...
+Github `issues <http://github.com/popgensims/stdpopsim/issues>`_.
 There is a lot of chatter through
 Github, and we’ve been building code
 there cooperatively. If you are interested in joining us, as we very much hope you
 are, please read our
 `code of conduct <https://github.com/popsim-consortium/stdpopsim/blob/master/CODE_OF_CONDUCT.md>`_.
 A particular goal of this project is to build a community which is representative of population
-and evolutionary genetics writ large. 
+and evolutionary genetics as a field. We especially encourage individuals from underrepresented groups
+in to join. Being a member of the Consortium can provide a valuable avenue for career development, through
```
Comment (agladstein) [inline docs/introduction.rst:55]: "from underrepresented groups in population genetics to join"

**Review by agladstein — COMMENTED** [review]
found a typo.

other than that looks good to me.

File: docs/introduction.rst (hunk 4 above)
Comment (andrewkern) [reply docs/introduction.rst:55]: 👍

**Review by agladstein — APPROVED** [review]

File: docs/introduction.rst [hunk 5]
```diff
@@ -35,11 +35,28 @@ First steps
...
 standards for benchmarking and testing in population genetics.
-Read the :ref:`Development <sec_development>` page to find out how you can join us.
+You can get into contact with the ``stdpopsim`` community by subscribing to our `email list
+serve <https://lists.uoregon.edu/mailman/listinfo/popgen_benchmark>`_
+and by creating and commenting on
+Github `issues <http://github.com/popgensims/stdpopsim/issues>`_.
+There is a lot of chatter through
+Github, and we’ve been building code
+there cooperatively. If you are interested in joining us, as we very much hope you
+are, please read our
```
Comment (petrelharp) [inline docs/introduction.rst:51]: cut "as we very much hope you are", as it sounds a bit pleading? We did already say "we welcome YOU to join us" up top.

File: docs/introduction.rst [hunk 6]
```diff
@@ -35,11 +35,28 @@ First steps
...
+`code of conduct <https://github.com/popsim-consortium/stdpopsim/blob/master/CODE_OF_CONDUCT.md>`_.
+A particular goal of this project is to build a community which is representative of population
+and evolutionary genetics as a field. We especially encourage individuals from underrepresented groups
+in popgen to join. Being a member of the Consortium can provide a valuable avenue for career development, through
```
Comment (petrelharp) [inline docs/introduction.rst:55]: These sentences clash a bit, as we say we want to represent popgen, but then ask for people underrepresented in popgen. How about just
"
A particular goal of this project is to build a diverse and supportive community, and so especially encourage individuals from underrepresented groups in popgen to join.
"

**Comment by petrelharp** [comment]
Minor suggestions: looks good.

**Review by petrelharp — APPROVED** [review]

File: docs/introduction.rst [hunk 7]
```diff
@@ -35,11 +35,27 @@ First steps
...
+``stdpopsim`` **is a community effort, and we welcome YOU to join us!**
+
 Are there other features, models or organisms that you'd like to see in ``stdpopsim``?
 This software is maintained by the PopSim Consortium,
-a global community of scientists and developers who are working together to improve
+an open, global community of scientists and developers who are working together to improve
 standards for benchmarking and testing in population genetics.
-Read the :ref:`Development <sec_development>` page to find out how you can join us.
+You can get into contact with the ``stdpopsim`` community by subscribing to our `email list
+serve <https://lists.uoregon.edu/mailman/listinfo/popgen_benchmark>`_
```
Comment (grahamgower) [inline docs/introduction.rst:45]: "email list serve". I guess this should be "server", but maybe just "email list" is sufficient?

**Review by grahamgower — APPROVED** [review]

**Pull request merged**